tgui/packages/tgui/interfaces/PrisonerManagement.tsx

```diff
@@ -0,0 +1,241 @@
+                  {button.name}
+                </Button>
+              ))}
+            </LabeledList.Item>
+          )}
+          <LabeledList.Divider />
+        </LabeledList>
+      </Stack.Item>
+    </Stack>
+  );
+};
+
+const AllImplantDisplay = (props: { implants: ImplantInfo[] }) => {
+  const implantsByCategory: Record<string, ImplantInfo[]> =
+    props.implants.reduce((acc, implant) => {
+      if (!acc[implant.category]) {
+        acc[implant.category] = [];
+      }
+      acc[implant.category].push(implant);
+      return acc;
+    }, {});
+
+  const [implantTab, setImplantTab] = useLocalState(
+    'implantTab',
+    Object.keys(implantsByCategory)[0]
+  );
+
+  // Converts a category ("tracking implant") to a more readable format ("Tracking")
+  const formatCategory = (category: string) => {
+    return category
+      .split(' ')
+      .map((word) => word.charAt(0).toUpperCase() + word.slice(1))[0];
+  };
+
+  return (
+    <Stack fill vertical>
+      <Stack.Item>
+        <Tabs>
+          {Object.entries(implantsByCategory).map(([category, implants]) => (
+            <Tabs.Tab
+              key={category}
+              selected={implantTab === category}
+              onClick={() => setImplantTab(category)}>
+              {formatCategory(category)}
+            </Tabs.Tab>
+          ))}
+        </Tabs>
+      </Stack.Item>
+      <Stack.Item>
+        {implantsByCategory[implantTab] &&
+          implantsByCategory[implantTab].map((implant) => (
+            <ImplantDisplay key={implant.ref} implant={implant} />
+          ))}
+      </Stack.Item>
+    </Stack>
+  );
+};
+
+const IdShowcase = (props: { id: IDInfo | null }) => {
+  const { act } = useBackend<IDInfo>();
+  const { id } = props;
+
+  return (
+    <Stack fill vertical>
+      <Stack.Item>
+        <LabeledList>
+          {id ? (
+            <>
+              <LabeledList.Item label="ID">
+                <Button onClick={() => act('eject_id')} icon="eject" mr={1} />
+                {id.name}
+              </LabeledList.Item>
+              <LabeledList.Item label="Points">
+                <Button
+                  onClick={() => act('reset_id')}
+                  icon="times"
+                  color="bad"
+                  mr={1}
+                />
+                {id.points}
+              </LabeledList.Item>
+              <LabeledList.Item label="Goal">
+                <Button onClick={() => act('set_id_goal')} icon="info" mr={1} />
+                {id.goal}
+              </LabeledList.Item>
+            </>
+          ) : (
+            <LabeledList.Item label="ID">
+              <Button onClick={() => act('insert_id')}>No ID Inserted</Button>
+            </LabeledList.Item>
+          )}
+        </LabeledList>
+      </Stack.Item>
+      {!!id && (
+        <Stack.Item>
+          <NoticeBox>
+            Space Law recommends quotas of 100 points per minute they would
+            normally serve in the brig.
```

Comment:
```suggestion
            Space Law recommends quotas of 100 points per minute that they would
            normally serve in the brig.
```
this reads weirdly, sorry

Reply:
This is copied directly from what the console currently says. 
Also, `per minute they would serve` sounds fine to me? I dunno.